virt/about-virt.adoc

```diff
@@ -20,4 +20,23 @@ You can check your {VirtProductName} cluster for compliance issues by installing
 :FeatureName: {VirtProductName} integration with the Compliance Operator
 include::snippets/technology-preview.adoc[]
 
-include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+[id="virt-snocaps-differences_{context}"]
+= Single Node {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster. See xref:../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] to prepare your cluster for installation.
+
+When provisioning an {product-title} {sno} cluster with the link:https://cloud.redhat.com/blog/using-the-openshift-assisted-installer-service-to-deploy-an-openshift-cluster-on-metal-and-vsphere[assisted installer], out-of-the-box persistent storage is auto-deployed.
+
+* In {VirtProductName} 4.10 and 4.11, the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Hostpath Provisioner (HPP)] is automatically installed.
```

Comment:
Valid point, but I think it's needed in this case so users understand how the implementation has changed compared to past releases. It will also prevent experienced users from becoming disoriented, as in "_what_ did they do to this feature....?"

Reply:
Fair enough then :)